portality/static/doaj/AccessIndicators.xsd

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For ease of understanding later, I'd suggest we put all the crossref XSDs in their own directory, like `static/crossref/*.xsd`